src/client/src/rt-components/platform/adapters/glue/glue.ts

```diff
@@ -0,0 +1,60 @@
+import { BasePlatformAdapter } from '../platformAdapter'
+import { WindowConfig } from '../types'
+import { openGlueWindow, registerWindowMethods } from './window'
+import { sendNotification, NotifyPermission } from './utils/sendNotification'
+import { CurrencyPairPositionWithPrice } from '../../../../rt-types/index'
+import { getAddinStatus, openSheet, updateSheet } from './utils/excel'
+const Glue = require('tick42-glue').default
+const Glue4Office = require('glue4office').default
+
+/**
+ * Glue implementation of the base platform adapter.
+ * Glue4Office is optional library for working with excel.
+ */
+Glue({ channels: true })
+  .then((glue: any) => {
```

Comment:
Glue42 should have type definitions. This can be done in Relative trader as at [file].d.ts file. An example of this would be `ExcelService.d.ts`.

You can also add glue42 to the window type via declaration merging.